doc/DirDiff.txt

```diff
@@ -0,0 +1,101 @@
+*DirDiff.vim*   A plugin to diff and merge two directories recursively

```

Comment:
Actually it would be better to implement an index, include headings and internal references (*DirDiff-options*, etc), as explained in `:help write-local-help`. This is just a starting point, which I find better than no help file at all :-)